.travis.yml

```diff
@@ -4,3 +4,19 @@ node_js:
   - '0.10'
   - '0.12'
   - 'iojs'
+
+cache:
+  directories:
+    - node_modules
```

Comment:
I don't really like this. It means the build may fail on a freshly cloned repo and still pass on Travis because a sub-dependency got updated and Travis didn't catch it.